actions/sync/entrypoint

```diff
@@ -37,11 +38,12 @@ sync_dirs() {
   set -x
   cd "${config_repo}${config_path}"
     for dir in $(ls -Ap . | grep "/"); do
-      if [[ -f "${current_repo}/${dir}.syncignore" ]]; then
-        rsync -r "${dir}" "${current_repo}/${dir}" --delete \
-          --exclude=".syncignore" --exclude-from="${current_repo}/${dir}.syncignore"
+      local exclude_list=""
+      if [[ -f "${current_repo}${current_path}/${dir}.syncignore" ]]; then
+        rsync -r "${dir}" "${current_repo}${current_path}/${dir}" --delete \
+          --exclude=".syncignore" --exclude-from="${current_repo}${current_path}/${dir}.syncignore"
       else
-        rsync -r "${dir}" "${current_repo}/${dir}" --delete
+        rsync -r "${dir}" "${current_repo}${current_path}/${dir}" --delete
```

Comment:
```suggestion
        rsync -r "${dir}" "${current_repo}/${current_path}/${dir}" --delete
```

Reply:
I think we don't want it just incase its an empty string?

Reply:
That's totally fine. Bash will see a path like `/some-org/some-repo//some-dir` and convert that to `/some-org/some-repo/some-dir`.